src/GameJQuery.js

```diff
@@ -0,0 +1,147 @@
+console.log("Atty helped with the JQuery")
+
+$( document ).ready(function() {
+
+  var game = new Game();
+  game.start(10);
+
+
+  //     for (var i = 1; i <= 10; i ++){
+  //       $('#r'+i).text(game.frames[i - 1].bowls[0]);
+  //       $('#r2').text(game.frames[i - 1].bowls[1]);
+  //       $('#r3').text(game.frames[i - 1].bowls[2]);
+  //     };
+  //   };
+
+  $( "#b0" ).click(function() {
+      var arrayFrame = game.currentFrame-1
+      var frame = game.currentFrame
+      var roll = (game.frames[arrayFrame].isRoll)
+      game.roll(0)
+      $( "#r" + (arrayFrame + 1) + roll ).text(game.frames[arrayFrame].bowls[roll-1]);
+      displaysScore(roll,frame)
+    });
+
+  $( "#b1" ).click(function() {
+      var arrayFrame = game.currentFrame-1
```

Comment:
DRY!

JQuery is kind of new and shiny, so a lot of people come unstuck here - but you can apply handler code like this to an array of elements using e.g. a class, and then for supplying the individual scores you have a few things you can do. One is just to examine the text of the button:

```
<button id="b0" class='score-button' type="button">0</button>
<button id="b1" class='score-button' type="button">1</button>
...
<button id="b10" class='score-button' type="button">10</button>

...

function handleFrameClick() {
  var score = parseInt($(this).text());
  var arrayFrame = game.currentFrame-1
  var frame = game.currentFrame
  var roll = (game.frames[arrayFrame].isRoll)
  game.roll(score)
  $( "#r" + (arrayFrame + 1) + roll ).text(game.frames[arrayFrame].bowls[roll-1]);
  displaysScore(roll,frame)
}

$('.score-button').click(handleClickFrame);
```

Alternatively, you could have an attribute on the 

```
<button id="b0" class='score-button' score='1' type="button">0</button>
...
var score = parseInt($(this).attr('score-button'));
```